std-bits/base/src/main/java/org/enso/base/numeric/Conversion_Result.java

```diff
@@ -0,0 +1,6 @@
+package org.enso.base.numeric;
+
+import java.math.BigDecimal;
+
+// Wraps the result of a conversion to BigDecimal with a precision loss flag.
+public record Conversion_Result(BigDecimal newValue, boolean hasPrecisionLoss) {}
```

Comment:
nitpick and we are not even consistent here throughout the codebase, but IMHO if it's a Java class it would better have its name be CamelCase
```suggestion
public record ConversionResult(BigDecimal newValue, boolean hasPrecisionLoss) {}
```

I know we do have Snake_Case Java classes as well, but I'm not sure if that's right that we do have them.

Regardless, at least within the class I think we should rather try to keep the naming consistent - i.e. if the fields are `camelCase` then the class probably should be too.

One more benefit of that is that later in Enso I can more clearly see that this is a Java type and not an Enso type (although I probably shouldn't have to know that in most cases, but it feels useful).

Reply:
Done